wiki/People/User_group/de.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - usergroup
+---
+
+# Benutzergruppe
+
+*Anmerkung: Die Auflistungen für die Benutzergruppen Chat-Bots und Ankündigungen sind nicht öffentlich.*
+
+![Beispiel für ein Gruppenabzeichen](img/user-group-badge.png "Gruppenabzeichen auf der Profilseite eines Nutzers")
+
+Im Folgenden sind **Benutzergruppen** aufgeführt, von denen die meisten aus Mitgliedern der osu!-Community bestehen, die helfen, osu! zu erhalten. Viele dieser Benutzergruppen sind anhand der jeweiligen Farben im [Forum](/wiki/Community/Forum), Farben im [Spiel-Chat](/wiki/Client/Interface/Chat_console), [Profiltiteln](/wiki/Community/User_title) und/oder Gruppenabzeichen erkennbar.
+
+Ein Klick auf die Abzeichen führt zu den jeweiligen Gruppenlisten, während die Namen auf die Wiki-Artikel verweisen.
+
+| ID | Abzeichen | Name | Beschreibung |
+| :-: | :-: | :-- | :-- |
+| 4 | [![GMT](/wiki/shared/group/GMT.png)](https://osu.ppy.sh/groups/4) | [Globales Moderationsteam](/wiki/People/Global_Moderation_Team) | Behalten die Übersicht über die Foren und den Spiel-Chat |
+| 7 | [![NAT](/wiki/shared/group/NAT.png)](https://osu.ppy.sh/groups/7) | [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) | Verwaltung der Beatmap Nominators |
+| 11 | [![DEV](/wiki/shared/group/DEV.png)](https://osu.ppy.sh/groups/11) | [Entwickler](/wiki/People/Developers) | Machen das Spiel besser, indem sie neue Inhalte zum Spiel hinzufügen und Fehler beheben |
+| 16 | [![ALM](/wiki/shared/group/ALM.png)](https://osu.ppy.sh/groups/16) | [osu! Alumni](/wiki/People/Developers) | Diejenigen, die für ihre Beiträge bekannt sind und inzwischen weitergezogen sind |
+| 22 | [![SPT](/wiki/shared/group/SPT.png)](https://osu.ppy.sh/groups/22) | [Support-Team](/wiki/People/Support_Team) | Hilfe und Unterstützung |
+| 28 | [![BN](/wiki/shared/group/BN.png)](https://osu.ppy.sh/groups/28) | [Beatmap Nominators](/wiki/People/Beatmap_Nominators) | Sorgen dafür, dass Beatmaps den Status "qualifiziert" erlangen |
+| 29 | ![BOT](/wiki/shared/group/BOT.png) | [Chat-Bots](/wiki/Bot_account) | Spezielle Accounts, die von automatisierten Diensten anstatt von echten Personen verwendet werden |
+| 31 | [![LVD](/wiki/shared/group/LVD.png)](https://osu.ppy.sh/groups/31) | [Project Loved](/wiki/People/Project_Loved_Team) | Erkennen der Beatmaps, die die Community am meisten liebt |
```

Comment:
```suggestion
| 31 | [![LVD](/wiki/shared/group/LVD.png)](https://osu.ppy.sh/groups/31) | [Project Loved](/wiki/People/Project_Loved_Team) | Auszeichnen der Beatmaps, die die Community am meisten liebt |
```
A possible alternative I thought about.

Reply:
```suggestion
| 31 | [![LVD](/wiki/shared/group/LVD.png)](https://osu.ppy.sh/groups/31) | [Project Loved](/wiki/People/Project_Loved_Team) | Anerkennen der Beatmaps, die die Community am meisten liebt |
```
Maybe this could also work, i think it stays more true to the meaning of `recognising`

Reply:
This sounds good as well ^^